cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -183,39 +231,31 @@ func generateMountsModeRecover(mst *initramfsMountsState, recoverySystem string)
 		return nil
 	}
 
-	// now copy the auth data from the real ubuntu-data dir to the ephemeral
-	// ubuntu-data dir
+	// 4. final step: copy the auth data from the real ubuntu-data dir to the
+	//    ephemeral ubuntu-data dir, write the modeenv to the tmpfs data, and
+	//    disable cloud-init in recover mode
 	if err := copyUbuntuDataAuth(boot.InitramfsHostUbuntuDataDir, boot.InitramfsDataDir); err != nil {
 		return err
 	}
-
-	// n+2: final step: write $(tmpfs-data)/var/lib/snapd/modeenv - this
-	//      is the tmpfs we just created above
 	modeEnv := &boot.Modeenv{
 		Mode:           "recover",
 		RecoverySystem: recoverySystem,
 	}
 	if err := modeEnv.WriteTo(boot.InitramfsWritableDir); err != nil {
 		return err
 	}
-	// and disable cloud-init in recover mode
 	if err := sysconfig.DisableCloudInit(boot.InitramfsWritableDir); err != nil {
 		return err
 	}
 
-	// n+3: done, no output, no error indicates to initramfs we are done
-	//      with mounting stuff
+	// done, no output, no error indicates to initramfs we are done with
+	// mounting stuff
 	return nil
 }
 
-var (
-	secbootMeasureSnapSystemEpochWhenPossible = secboot.MeasureSnapSystemEpochWhenPossible
-	secbootMeasureSnapModelWhenPossible       = secboot.MeasureSnapModelWhenPossible
-	secbootUnlockVolumeIfEncrypted            = secboot.UnlockVolumeIfEncrypted
-)
-
 func generateMountsCommonInstallRecover(mst *initramfsMountsState, recoverySystem string) (allMounted bool, err error) {
-	// 1. always ensure seed partition is mounted
+	// 1.1. always ensure seed partition is mounted first before the others,
```

Comment:
There's no 1.2, should this be just 1?

Reply:
oh whoops I think I had a 1.2 at a certain point during refactoring but forgot to move it back, good catch